lib/more_core_extensions/core_ext/string/iec60027_2.rb

```diff
@@ -0,0 +1,17 @@
+module MoreCoreExtensions
+  module IEC60027_2
+    # Support with IEC size format
+    # http://physics.nist.gov/cuu/Units/binary.html
+    IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti Pi Ei Zi Yi).freeze
+    def iec60027_2
```

Comment:
I'm thinking this should say to_i or to_??_integer... because it handles a very specific conversion from String to that type...trying to think of how we do it for other things in manageiq, but there really isn't a precedent there